tests/failing/brils/float/float_args.bril

```diff
@@ -1,4 +0,0 @@
-# ARGS: 3 3.0 -0.3
```

Comment:
This `print` takes several arguments. Should we support it?

Reply:
Seems okay to skip